src/local-settings.js

```diff
@@ -59,16 +60,21 @@ const CONFIGURATION_VARS = [
   [PORTIS_DAPP_ID, process.env.ARAGON_PORTIS_DAPP_ID],
   [FORTMATIC_API_KEY, process.env.ARAGON_FORTMATIC_API_KEY],
 ].reduce(
-  (acc, [option, envValue, envValueCompat]) => ({
+  (acc, [option, ...envValues]) => ({
     ...acc,
     [option]: {
       storageKey: `${option}_KEY`,
-      envValue: envValue || envValueCompat || null,
+      envValue: envValues.reduce((prevVal, envValue) => prevVal || envValue),
     },
   }),
   {}
 )
 
+// Get a search param from the location's hash
+function getUrlParam(paramName) {
+  return new URLSearchParams(window.location.hash.split('?')[1]).get(paramName)
+}
```

Comment:
This should go in `url-utils.js`; I also believe we might want to hold off until [`useRouting()`](https://github.com/aragon/aragon/pull/1272) has been merged so this does not get outdated.

Reply:
PR https://github.com/aragon/aragon/pull/1272 has been opened for many months and this is critical for cross development of Aragon Apps that use a setup slightly different than the default. Is there a way to find a way to customize the ENS address that could be merged into the client?